adserver/models.py

```diff
@@ -1186,10 +1193,12 @@ def views_needed_this_interval(self):
 
     def clicks_needed_this_interval(self):
```

Comment:
Do we need to update `views_needed_this_interval` as well?

Reply:
Yes it does.